dali/operators/generic/slice/slice_attr.h

```diff
@@ -53,11 +56,17 @@ class SliceAttr {
     // slice args are CPU inputs
     const auto &crop_anchor = ws.template Input<CPUBackend>(1);
     const auto &crop_shape = ws.template Input<CPUBackend>(2);
+    DALI_ENFORCE(crop_anchor.type().id() == crop_shape.type().id(),
+                  make_string("Anchor and shape should have the same type. Got: ",
+                              crop_anchor.type().id(), " and ", crop_shape.type().id()));
```

Comment:
Would this `crop_anchor.type().id()` be converted to a meaningful string?

Reply:
yes